src/AppBundle/Entity/ChangeLogItem.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * ChangeLogItem
+ *
+ * @ORM\Table(name="change_log_item")
+ * @ORM\Entity(repositoryClass="AppBundle\Entity\Repository\ChangeLogItemRepository")
+ */
+class ChangeLogItem
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="title", type="string",nullable=false)
```

Comment:
Siden du asserter max length her, kunne vi kanskje lagt på en database constraint for dette og?
`@ORM\Column(type="string", length=40, nullable=false)`